source/portable/NetworkInterface/STM32Fxx/stm32fxx_hal_eth.h

```diff
@@ -45,11 +45,11 @@
     #define __STM32F7xx_HAL_ETH_H
 
     #if defined( STM32F7xx )
-        #include "stm32f7xx_hal_def.h"
+        #include "stm32f7xx_hal.h"

```

Comment:
/md
Indeed, that change is necessary, if not there will be compiler errors like `#error please define ETH_RX_BUF_SIZE`

Reply:
Ping ping